packages/bpk-component-navigation-stack/README.md

```diff
@@ -34,7 +34,7 @@ that can be used to push and pop views from the stack component.
 
 ```js
 import React from 'react';
-import BpkNavigationStack from 'bpk-component-navigation-stack';
+import BpkNavigationStack, { withNavigationStackState } from 'bpk-component-navigation-stack';
```

Comment:
It's quite obvious how it should be imported, but good to make it explicit anyway 😅